src/lib/posthog-events.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/**
+ * Enables PostHog video start tracking
+ */
+export function trackVideoStart(url: string) {
+	if (!window?.posthog) return
+
+	const video_host = url.includes('wistia')
+		? 'wistia'
+		: url.includes('youtube')
+		? 'youtube'
+		: 'other'
```

Comment:
to make this easier to read for the next dev
```suggestion
	let video_host = ""
	if (url.includes('wistia')) {
		video_host = 'wistia'
	} else if (url.includes('youtube')) {
		video_host = 'youtube'
	} 
```